### PR TITLE
Update schools.md

### PR DESCRIPTION
**Pull request by mixedfruitjuice**
I added some eventual high schools/colleges from which people could be participating in Coffee and Coding

### CODE REVIEW

File: schools.md [hunk 1]
```diff
@@ -276,6 +277,7 @@ Gujarat University
 Guru Gobind Singh Indraprastha University
 Hampshire College
 Hampton University
+Hanze university of applied sciences
```
Comment (theycallmeswift) [inline schools.md]: Need to capitalize University, Applied, and Sciences.

File: schools.md [hunk 2]
```diff
@@ -702,6 +704,7 @@ Spotswood High School
 Spring Arbor University
 Sreenidhi Institute of Science & Technology
 Sri Sivasubramaniya Nadar College of Engineering
+Stadslyceum
```
Comment (theycallmeswift) [inline schools.md]: The official name of the school may be "The Stadslyceum".  Tough to tell from the website

File: schools.md [hunk 3]
```diff
@@ -1086,4 +1090,4 @@ Zespół szkół nr 1 im. Stanisława Staszica w Bochni
 Zespół Szkół im. Jana Pawła II w Niepołomicach
 École Centrale Paris
 École Polytechnique de Montréal
-École de technologie supérieure
\ No newline at end of file
+École de technologie supérieure
```
Comment (theycallmeswift) [inline schools.md]: Need to capitalize Technologie and Supérieure

**Comment by jonmarkgo** [comment]
Merged here. https://github.com/MLH/mlh-policies/commit/f907e649cf41d5a2316ad250b3687c717e93076c

I did HN Werkman instead of Stads since that is just the name of a building

**Pull request closed without merging**